imap-codec/Cargo.toml

```diff
@@ -3,7 +3,7 @@ name = "imap-codec"
 description = "Rock-solid and complete codec for IMAP"
 keywords = ["email", "imap", "codec", "parser"]
 categories = ["email", "parser-implementations", "encoding", "network-programming"]
-version = "2.0.0-alpha.3"
+version = "2.0.0-alpha.4"
```

Comment:
interestingly, we don't need to publish `imap-types 2.0.0-alpha.4`, too. We could let the version go out-of-sync... Which is... good?

Reply:
I think that's okay. What do we do the next time when we need to publish both crates? Bump both crates to the same version?

Reply:
Let's discuss next time :D Not sure... We could also just let it be out-of-sync.